flepimop/gempyor_pkg/src/gempyor/inference_parameter.py

```diff
@@ -53,8 +53,8 @@ def add_modifier(self, pname, ptype, parameter_config, subpops):
                     pname=pname,
                     subpop=sp,
                     pdist=parameter_config["value"].as_random_distribution(),
-                    lb=parameter_config["value"]["a"].get(),
-                    ub=parameter_config["value"]["b"].get(),
+                    lb=float(parameter_config["value"]["a"].get()),
```

Comment:
Is there a specific reason this is of a different form to lines 68 and 69? Otherwise I think this will deal with the error. thanks!

Reply:
Oops nope, just fixed them so that they match. Good catch